app/src/main/java/com/incwelltechnology/lms/data/repository/UserRepository.kt

```diff
@@ -0,0 +1,31 @@
+package com.incwelltechnology.lms.data.repository
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import com.incwelltechnology.lms.data.model.LoginRequest
+import com.incwelltechnology.lms.data.model.LoginResponse
+import com.incwelltechnology.lms.data.network.LmsApi
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class UserRepository {
```

Comment:
@iamsannyrai why do we need user repository?

Reply:
User repository communicates with AuthViewModel (viewmodel) and viewmodel with UI(activity or fragment), so to make clear distinction of our concerns, i have created UserRepository